### PR TITLE
feat: lending support all thor-supported assets as borrow

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

vs. ETH and BTC only previously

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes N/A

## Risk

The assets list might be broken in "Available Pools" and "Your loans"

## Testing

- All THOR-supported assets are available as assets to borrow
- All THOR-supported assets are still available as assets to repay with

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

### Borrow available assets

Screenshot 2023-11-22 at 00 25 12
Screenshot 2023-11-22 at 00 25 12

### Repay available assets

image
image

### Available pools

image

### Your loans

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5675** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5675?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5675?utm_source=stack-comment).

File: src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx [hunk 1]
```diff
@@ -239,10 +239,11 @@ export const BorrowInput = ({
     history.push(BorrowRoutePaths.Sweep)
   }, [history, isSweepNeeded])
 
-  const depositAssetSelectComponent = useMemo(() => {
+  const collateralAssetSelectComponent = useMemo(() => {
     return (
       <TradeAssetSelect
         assetId={collateralAssetId}
+        // no-op
```
Comment (kaladinlight) [inline src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx:246]: ?

Comment (gomesalexandre) [reply src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx:246]: Actually a no-op, we can't change the collateral for borrows

File: src/pages/Lending/hooks/useLendingSupportedAssets.ts [hunk 2]
```diff
@@ -42,6 +30,24 @@ export const useLendingSupportedAssets = () => {
         })
         .filter(isSome)
     },
+    [type],
+  )
+
+  const lendingPositionData = useQuery({
```
Comment (gomesalexandre) [inline src/pages/Lending/hooks/useLendingSupportedAssets.ts:36]: ser

Comment (gomesalexandre) [reply src/pages/Lending/hooks/useLendingSupportedAssets.ts]: c425a85bfa8b4d0340c74ea8c196afd8854157f2

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**